app/frontend/Calendar/DisplayEvent/DialogHeader.svelte

```diff
@@ -132,7 +132,8 @@
   async function onDeleteRemainder() {
     event.parentEvent.cancelEditing();
     event.cancelEditing();
-    await event.truncateRecurrence(event.startTime);
+    event.truncateRecurrencesFromHere();
+    await event.save();
```

Comment:
You don't want to save the event here as that tries to turn it from an instance into an exception. (The old `truncateRecurrence` had `await master.save()`, which is why it was `async`.)
```suggestion
    event.truncateRecurrencesFromHere();
    await event.parentEvent.save();
```